src/components/app/userActionRowCTA/userActionRowCTAsList.tsx

```diff
@@ -8,15 +8,19 @@ import { USER_ACTION_ROW_CTA_INFO } from '@/components/app/userActionRowCTA/cons
 import { cn } from '@/utils/web/cn'
 import { USER_ACTION_TYPE_CTA_PRIORITY_ORDER } from '@/utils/web/userActionUtils'
 
+export interface UserActionRowCTAsListProps {
+  performedUserActionTypes?: UserActionType[]
+  excludeUserActionTypes?: UserActionType[]
+  className?: string
+  render?: (props: React.ComponentPropsWithoutRef<typeof UserActionRowCTA>) => React.ReactNode
```

Comment:
This should be a component, doing it like this you're removing a part of the lifecycle ou of it (see the 2 other comments bellow for the implementation)
```suggestion
  render?: React.ComponentType<React.ComponentPropsWithoutRef<typeof UserActionRowCTA>>
```